blogposts/announcing-sourcegraph-3.10.md

```diff
@@ -0,0 +1,198 @@
+</p>
+-->
+
+Sourcegraph now supports creating and tracking code-aware search and replace campaigns on GitHub and Bitbucket Server. When running a search and replace campaign, you preview the changes, then generate the corresponding pull requests on your GitHub and/or Bitbucket Server instances. Once created, you can track the PRs to completion in a burndown chart.
+
+Sourcegraph is introducing [Comby syntax](https://comby.dev/#basic-usage) for automation search and replace campaigns. Comby is a code aware search syntax that simplifies finding patterns in code. This makes operations that previously required complex regexp or sed queries much simpler to write. For example, changing how errors are logged througout your code:
+
+```c
+// match:
+errors.New(fmt.Sprintf(:[args]))
+
+// replace:
+fmt.Errorf(:[args])
+```
+
+Sourcegraph Automation campaigns are in private beta. [Watch the Automation screencasts](https://about.sourcegraph.com/product/automation#see-it-in-action) to see what we have planned and [apply for early access](https://about.sourcegraph.com/contact/request-automation-demo/) to Automation for your organization.
+
+## Precise LSIF-based code intelligence for 5 languages
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+With the completion of the Java LSIF indexer, LSIF-based precise code intelligence now supports five languages: Go, Typescript, Java, C++, and Python. See [lsif.dev](https://lsif.dev/) for the most up-to-date information on the list of languages with LSIF indexers.
+
+We are looking for feedback from project owners interested in trying out precise code intelligence! If you are responsible for projects using any of the languages we support, we’d love to hear from you. Use our new [LSIF quickstart guide](https://docs.sourcegraph.com/user/code_intelligence/lsif_quickstart) to try it out manually before adding it as a step in your CI.
+
+## Scaling search for our largest customers
+
+Customers with large Kubernetes deployments can now horizontally scale their search indexes by adding more search nodes.
+
+**This improvement requires a Kubernetes migration**. The Kubernetes [deployment manifest](https://github.com/sourcegraph/deploy-sourcegraph) for indexed-search services has changed from a Normal Service to a Headless Service. This enables Sourcegraph to individually resolve indexed-search pods. Services are immutable, so please follow the [migration guide](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md#310).
```

Comment:
This needs to be much more front-and-center (maybe out of this section and more near the top or something?). It's not just people wanting this improvement that need a migration -- ALL Kubernetes clusters will need to run this migration, period. I would phrase it something like:

> **IMPORTANT:** Kubernetes clusters will require a manual migration, please see [the migration guide](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md#310) for details.

Reply:
How's this?
![image](https://user-images.githubusercontent.com/1559622/69210263-01277700-0b0f-11ea-9060-0d0e02e11706.png)